pkg/api/v1alpha1/tinkerbellmachineconfig_types_test.go

```diff
@@ -67,7 +67,7 @@ func TestTinkerbellMachineConfigValidateFail(t *testing.T) {
 
 type tinkerbellMachineConfigOpt func(mc *TinkerbellMachineConfig)
 
-func createTinkerbellMachineConfig(options ...tinkerbellMachineConfigOpt) *TinkerbellMachineConfig {
+func CreateTinkerbellMachineConfig(options ...tinkerbellMachineConfigOpt) *TinkerbellMachineConfig {
```

Comment:
curious why make this exported?

Reply:
we use the same method in `tinkerbellmachineconfig_types_test` and `tinkerbellmachineconfig_webhook_test`